toolchain/sem_ir/node_kind.def

```diff
@@ -7,101 +7,64 @@
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(ConstType, "const_type", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(Dereference, "dereference", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(FunctionDeclaration, "fn_decl", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(InitializeFrom, "initialize_from", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(IntegerLiteral, "int_literal", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(NameReference, "name_reference", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(Namespace, "namespace", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(NoOp, "no_op", None, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(Parameter, "parameter", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(PointerType, "ptr_type", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(RealLiteral, "real_literal", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(ReturnExpression, "return", None, Terminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(Return, "return", None, Terminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(SpliceBlock, "splice_block", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(StringLiteral, "string_literal", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(StructAccess, "struct_access", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(StructInit, "struct_init", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(StructLiteral, "struct_literal", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(StructTypeField, "struct_type_field", None,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(StructType, "struct_type", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(StructValue, "struct_value", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(TemporaryStorage, "temporary_storage", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(Temporary, "temporary", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(TupleAccess, "tuple_access", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(TupleIndex, "tuple_index", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(TupleInit, "tuple_init", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(TupleLiteral, "tuple_literal", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(TupleType, "tuple_type", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(TupleValue, "tuple_value", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(UnaryOperatorNot, "not", Typed, NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(ValueAsReference, "value_as_reference", Typed,
-                             NotTerminator)
-CARBON_SEM_IR_NODE_KIND_IMPL(VarStorage, "var", Typed, NotTerminator)
+CARBON_SEM_IR_NODE_KIND(AddressOf)
```

Comment:
```suggestion
// These have a 1:1 correspondence to entries in typed_nodes.h.
CARBON_SEM_IR_NODE_KIND(AddressOf)
```

Maybe add a comment in this file noting where to look for the rest?

Reply:
Done.